applications/EmpireApplication/tests/fsi_potential_flow_sdof/small_lift_jump_parameters.json

```diff
@@ -0,0 +1,71 @@
+{
+    "problem_data"                     : {
+        "problem_name"    : "naca0012",
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0,
+        "start_time"      : 0.0,
+        "end_time"        : 1
+    },
+    "solver_settings"                  : {
+        "solver_type"            : "potential_flow",
+        "domain_size"     : 2,
+        "model_part_name" : "FluidModelPart",
+        "model_import_settings"  : {
+            "input_type"     : "mdpa",
+            "input_filename" : "naca0012_small"
+        },
+        "maximum_iterations"     : 2,
+        "echo_level"             : 0,
+        "linear_solver_settings"  : {
+                "solver_type"             : "ExternalSolversApplication.super_lu"
+        },
+        "volume_model_part_name" : "Parts_Parts_Auto1",
+        "skin_parts"             : ["PotentialWallCondition2D_Far_field_Auto1","Body2D_Body"],
+        "no_skin_parts"          : []
+    },
+    "processes" : {
+        "initial_conditions_process_list"  : [],
+        "boundary_conditions_process_list" : [{
+            "python_module" : "apply_far_field_process",
+            "kratos_module" : "KratosMultiphysics.CompressiblePotentialFlowApplication",
+            "process_name"  : "FarFieldProcess",
+            "Parameters"    : {
+                "model_part_name"   : "FluidModelPart.PotentialWallCondition2D_Far_field_Auto1",
+                "inlet_phi"         : 1.0,
+                "velocity_infinity" : [10.0,0.0,0.0]
+            }
+         },{
+            "python_module" : "define_wake_process_2d",
+            "kratos_module" : "KratosMultiphysics.CompressiblePotentialFlowApplication",
+            "process_name"  : "DefineWakeProcess",
+            "Parameters"    : {
+                "model_part_name" : "FluidModelPart.Body2D_Body",
+                "wake_direction"       : [10.0,0.0,0.0],
+                "epsilon"         : 1e-9
+            }
+        },{
+            "python_module" : "compute_lift_jump_process_2d",
```

Comment:
This process does not exist any more, I have the feeling you are not using this parameters